client/v2/common/models/shim.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+// Types in this file are used to help convert between hand written code
+// that doesn't quite match the spec, and code generated from the spec.
```

Comment:
This was going to be too ugly to cram into the codegen. The generator maps things like `NodeStatusResponse` -> `NodeStatus` and this file connects the custom name to the generated types.